abs.yaml

```
@@ -3,4 +3,15 @@
 # which conda thinks aren't required
 build_parameters:
   - "--suppress-variables"
-  - "--skip-existing"
+  - "--error-overlinking"
+#  - "--skip-existing"
```

```suggestion
  - "--skip-existing"
```
Minor comment, if you do end up committing changes, skip-existing would be preferred.

---

do you mean setting skip CI? Nothing's uploaded, or will be uploaded before the PR's merged, so AFAIU skip-existing wouldn't do much?

---

not having skip-existing is good when you configure a dev channel, but gives you the risk of uploading an existing version when it is not set.

---

added back